devicon.json

```diff
@@ -388,6 +388,25 @@
         "color": "#e25a1c",
         "aliases": []
     },
+    {
+        "name": "apl",
+        "altnames": ["A Programming Language"],
+        "tags": [
+            "programming",
+            "language"
```

Comment:
```suggestion
            "language",
            "open-source",
            "cross-platform"
```

Reply:
Looks good to me! ✔
Committing changes!